src/test/java/fr/paris/lutece/portal/service/daemon/DaemonSchedulerTest.java

```diff
@@ -563,36 +568,38 @@ private void testEnqueueCoalesce( boolean shouldThrow ) throws ClassNotFoundExce
             executingDaemon.waitForCompletion( );
             inqueueDaemon.go( );
             inqueueDaemon.waitForCompletion( );
-            try
-            {
-                inqueueDaemon.go( 200, TimeUnit.MILLISECONDS );
-                fail( "Daemon started twice but should have been coalesced" );
-            }
-            catch( TimeoutException e )
-            {
 
-            }
+            inqueueDaemon.go( 200, TimeUnit.MILLISECONDS );
+            fail( "Daemon started twice but should have been coalesced" );
+
+
+        }
+        catch( TimeoutException e1 )
+        {
+            //ok;
+        }
+        catch( InterruptedException | BrokenBarrierException e )
+        {
+            fail("failed" + e + ":" + e.getMessage( ) );
```

Comment:
here you swallow the exception and you loose the stack trace which might be helpful. Why not let tests throw exceptions, which makes them fail ?